src/av2/geometry/se3.py

```diff
@@ -63,20 +67,48 @@ def transform_point_cloud(self, point_cloud: NDArrayFloat) -> NDArrayFloat:
 
         Args:
             point_cloud: Array of shape (N, 3). If the transform represents dst_SE3_src,
-                then point_cloud should consist of points in frame `src`
+                then point_cloud should consist of points in frame `src`.
 
         Returns:
-            Array of shape (N, 3) representing the transformed point cloud, i.e. points in frame `dst`
+            Array of shape (N, 3) representing the transformed point cloud, i.e. points in frame `dst`.
         """
         return self.transform_from(point_cloud)
 
+    def transform_sweep_from(self, sweep: Sweep) -> Sweep:
+        """Apply the SE(3) transformation to a lidar sweep object.
+
+        This operation will transform 3D points in the sweep to a new reference frame.
+        Example usage:
+            sweep_city = city_SE3_ego.transform_sweep_from(sweep_ego)
```

Comment:
```suggestion
            sweep_dst = dst_SE3_src.transform_sweep_from(sweep_src)
```

A bit more general and aligns with the docstring comment.